index.html

```diff
@@ -27,6 +27,8 @@
         <!--[if lt IE 8]>
             <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
         <![endif]-->
+        <!-- Adding some new improvements to navigation! -->
+        <!--  Navs are soooo cooooool! -->
     <nav class="navbar navbar-inverse navbar-fixed-top" role="navigation">
```

Comment:
Consider changing the font color for the <nav> links to white because the default gray included with the <navbar-inverse> may be difficult for users to identify clearly.